vectordatasource/transform.py

```diff
@@ -4577,6 +4657,25 @@ def _sort_network_gb(network, ref):
     return network_code * 10000 + min(ref, 9999)
 
 
+def _sort_network_pl(network, ref):
+    if network is None:
+        network_code = 9999
+    elif network == 'PL:motorway':
+        network_code = 0
+    elif network == 'PL:expressway':
+        network_code = 1
```

Comment:
motorway sorts after expressway in Poland? I must be getting my English translations scrambled ;)

Reply:
Indeed! I thought that was confusing too, but [motorways are a more important class of road in Poland than expressways](https://en.wikipedia.org/wiki/Highways_in_Poland#Definitions_of_highways_according_to_Polish_law). I think a closer translation into US terms, although not very close, would be that Polish motorways are like US Interstates (or British motorways), and Polish expressways are like US State highways (or British A-roads).